build-system/tasks/visual-diff.rb

```diff
@@ -382,9 +382,13 @@ def snapshot_webpages(page, webpages, config)
     forbidden_css = webpage['forbidden_css']
     loading_incomplete_css = webpage['loading_incomplete_css']
     loading_complete_css = webpage['loading_complete_css']
+    snapshot_delay_sec = webpage['snapshot_delay_sec']
     enable_experiments(page, webpage['experiments'])
     log('verbose', 'Navigating to page ' + yellow(url) + '...')
     page.visit(url)
+    if snapshot_delay_sec
+      sleep(snapshot_delay_sec)
```

Comment:
Convert the string to an integer before calling `sleep`:
```
  sleep(snapshot_delay_sec.to_i)
```

Can you test this locally with different values to make sure that the code works as desired? (To speed things up, you can locally skip all other tests and run just the one you're adding.)